crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -346,6 +358,25 @@ impl CheckpointStore {
         self.checkpoint_content.insert(contents.digest(), &contents)
     }
 
+    pub fn insert_verified_checkpoint_contents(
+        &self,
+        checkpoint: &VerifiedCheckpoint,
+        full_contents: VerifiedCheckpointContents,
+    ) -> Result<(), TypedStoreError> {
+        let full_contents = full_contents.into_inner();
+        self.full_checkpoint_content
+            .insert(checkpoint.sequence_number(), &full_contents)?;
+        let contents = full_contents.into_checkpoint_contents();
+        self.checkpoint_content.insert(contents.digest(), &contents)
```

Comment:
Do you think it'd be worth changing this to call `self.insert_checkpoint_contents` so it stays in sync if we ever make that function do more than this one line?